app/steps/ui/will/newleft/index.js

```diff
@@ -0,0 +1,21 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/left.json');
+module.exports = class NewWillLeft extends ValidationStep {
```

Comment:
Please export the module at the bottom of the file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#module-exports for an example